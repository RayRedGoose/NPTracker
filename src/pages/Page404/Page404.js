import './Page404.scss'
import React from 'react'

const Page404 = () => {
  return (
    <main className="page404">
      <h1>Please, enter correct url!</h1>
    </main>
  )
}

export default Page404
