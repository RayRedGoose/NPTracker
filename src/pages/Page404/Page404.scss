@import "../../variables.scss";

.page404 {
  h1 {
    color: $accent-color-primary-dark;
  }
}
