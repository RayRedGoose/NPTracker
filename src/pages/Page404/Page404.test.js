import React from 'react'
import { shallow } from 'enzyme'
import Page404 from './Page404'

it("should match snapshot with all data passed in correctly", () => {
  const page = shallow(<Page404 />)
  expect(page).toMatchSnapshot()
})
