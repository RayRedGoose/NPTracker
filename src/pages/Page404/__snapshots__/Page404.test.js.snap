// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with all data passed in correctly 1`] = `
<main
  className="page404"
>
  <h1>
    Please, enter correct url!
  </h1>
</main>
`;
